include/mbedtls/bignum.h

```diff
@@ -615,13 +615,13 @@ int mbedtls_mpi_mod_int( mbedtls_mpi_uint *r, const mbedtls_mpi *A, mbedtls_mpi_
  * \param X        Destination MPI
  * \param A        Left-hand MPI
  * \param E        Exponent MPI
- * \param N        Modular MPI
+ * \param N        Modular MPI, must be odd
```

Comment:
@yanesca I'm afraight I don't understand what you mean. Where do you want this restriction on the parameter be documented instead?